README.md

```diff
@@ -56,9 +56,17 @@ export default function ({ assets = [] }) {
 
 Why return a function like this? The exported function is passed into a Promise chain, which allows you to perform some async setup for your app before we try to call it to render any html.
 
-**Client.js**
+#### Client.js
+
+You can put pretty much anything in your client file, builder wont try to require and run it. It is however recommended that you use [Boiler Room Runner](https://github.com/everydayhero/boiler-room-runner#in-your-clientjs-file). It's got what you need.
```

Comment:
"builder" to BRB.

Let's move the recommendation for BRR out to it's own area maybe.